javascript.md

```diff
@@ -33,6 +33,8 @@ JavaScript CodeStyle
   - [Function Context](#function-context)
   - [Comments](#comments)
   - [Classes](#classes)
+  - [ECMAScript 6](#ecmascript-6)
```

Comment:
Yep, sorry. 
But I think the section should be named as ECMAScript 2015, because next name of standard will be ECMAScript 2016 etc.
(http://www.ecma-international.org/news/Publication%20of%20ECMA-262%206th%20edition.htm)